Automattic/wp-calypso
# Editor: Fix drag and drop media uploads.

This is a quick fix for #23208, which was introduced #21734.

**To Test:**
- Start a post.
- Drag an image into the editor area to upload it to the post.
- Verify that the image is uploaded correctly.
- Separately, write some text.
- Highlight the text and upload/insert a document.
- Verify that the text links to the uploaded document and isn't changed.
- Highlight new text and upload/insert multiple documents.
- Verify that the highlighted text is replaced with linked document names.
